secp-bouncy/src/main/java/org/bitcoinj/secp/bouncy/BouncyPubKey.java

```diff
@@ -47,15 +47,16 @@ public byte[] getEncoded() {
         return point.getEncoded(false);  // This has a prefix byte
     }
 
-    @Override
-    public String toString() {
-        return toStringDefault();
-    }
-
     @Override
     public ECPoint getW() {
         return new ECPoint(
                 point.normalize().getAffineXCoord().toBigInteger(),
                 point.normalize().getAffineYCoord().toBigInteger());
     }
+
+    @Override
+    public String toString() {
+        return toStringDefault();
+    }
+
```

Comment:
A surplus empty line.

Reply:
The empty line is fixed in a later commit in this series.